packages/assets/profiles/microsoft/microsoft-045e-065d.json

```diff
@@ -0,0 +1,112 @@
+{
+    "profileId" : "microsoft-045e-065d",
```

Comment:
Why not use "samsung-odyssey" instead of the VID/PID? That's more human-readable and the convention in this repo seems to be that we should use readable names.

Reply:
Good question!

This naming directly flows from the WinMR Gamepad.id values used in the WebVR era as keys for render model CDNs - for example: https://github.com/BabylonJS/Babylon.js/tree/master/assets/meshes/controllers/microsoft/045E-065D

The key question now is whether

Reply:
OpenXR as a layer underneath WebXR is moving away from directly exposing things like VID/PID as output, instead having apps negotiate the set of controllers (OpenXR interaction profiles) that they already know about.

Perhaps it’s OK for the UA in that world to inherit the new controller from the registry only the next time the UA revs.  In the meantime, OpenXR underneath will tell the UA that the user is using whichever UA-known controller is most similar to the user’s true controller.

Reply:
This also brings up a further possibility to drive cross-UA conformance for UAs built atop OpenXR.  Another package of JSON mappings could map each WebXR input profile to its corresponding OpenXR interaction profile, defining which button/axis index maps to which OpenXR `/user` path.  OpenXR-based UAs could then simply merge in the latest OpenXR mappings once per release to inherit conformant support for all the latest controllers.

Reply:
@thetuvix...so many words... o_O
Let me respond to each comment block individually:
* As these strings were the ones Microsoft chose to expose in the WebVR timeframe, I will be merging this pull request as-is unless you (as the hardware owner) specifically tell me to use different ones.
* As you suggest, when new OpenXR hardware comes out that a UA yet isn't aware of, the UA may be able to intuit which of the generic profiles to report
* Please file a separate issue to discuss tracking consistent OpenXR mappings if you'd like to track that specifically.

Reply:
Sounds good - the discussion in this thread is orthogonal to your refactoring.

Reply:
Filed 2 new issues here.

Reply:
Thanks!